apps/web/pages/api/trpc/[trpc].ts

```diff
@@ -47,17 +49,21 @@ export default trpcNext.createNextApiHandler({
       );
     }
 
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore ctx.req is available for SSR but not SSG
-    if (!!ctx?.req && allPublic && allOk && isQuery && !isThereANonCacheableQuery) {
+    if (!!ctx?.req && allOk && isQuery) {
       // cache request for 1 day + revalidate once every 5 seconds
       const ONE_DAY_IN_SECONDS = 60 * 60 * 24;
+      const timezone = z.string().safeParse(ctx.req.headers["x-cal-timezone"]);
```

Comment:
Shouldn't this be like this?

```suggestion
      const timezone = z.string().safeParse(ctx.req.headers["x-vercel-ip-timezone"]);
```